### PR TITLE
Flutterize

### PR DESCRIPTION
**Pull request by abarth**
Rename `dart:sky` to `dart:ui` and rename `package:sky` to `package:flutter`.

### CODE REVIEW

**Comment by abarth** [comment]
TBR @Hixie

File: examples/BUILD.gn [hunk 1]
```diff
@@ -7,7 +7,7 @@ group("examples") {
 
   deps = [
     "//examples/demo_launcher",
-    "//examples/fitness",
+    #"//examples/fitness",
```
Comment (Hixie) [inline examples/BUILD.gn]: ?

File: examples/address_book/pubspec.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 dependencies:
-  sky: any
+  flutter: any
   sky_tools: any
 dependency_overrides:
   material_design_icons:
     path: ../../sky/packages/material_design_icons
-  sky:
+  flutter:
     path: ../../sky/packages/sky
```
Comment (Hixie) [inline examples/address_book/pubspec.yaml]: Shouldn't this be sky/packages/flutter?

File: sky/engine/bindings/sky_internals.dart [hunk 3]
```diff
@@ -1,13 +0,0 @@
-// Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (Hixie) [inline sky/engine/bindings/sky_internals.dart]: Did you mean to check in a ui_internals.dart?

File: examples/BUILD.gn (hunk 1 above)
Comment (abarth) [inline examples/BUILD.gn]: I need to update playfair to understand the new package names.  This circular dependency is no good.

File: sky/engine/bindings/sky_internals.dart (hunk 3 above)
Comment (abarth) [inline sky/engine/bindings/sky_internals.dart]: Yes.  Will add shortly.

File: examples/BUILD.gn (hunk 1 above)
Comment (Hixie) [inline examples/BUILD.gn]: k. Add a TODO or link to an issue or something so there's a breadcrumb trail explaining what this mysteriously commented-out line means. :-)

Comment (abarth) [inline examples/BUILD.gn]: I should be able to get it fixed up before Monday.

**Pull request merged**